### PR TITLE
OTT-1300: fix label used for 'ow_pbs_endpoint_requests" metric

### PR DESCRIPTION
**Pull request by ashishshinde-pubm**

### CODE REVIEW

**Review by Pubmatic-Supriya-Patil — APPROVED** [review]
LGTM

File: modules/pubmatic/openwrap/beforevalidationhook.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func (m OpenWrap) handleBeforeValidationHook(
 	rCtx.Source, rCtx.Origin = getSourceAndOrigin(payload.BidRequest)
 	rCtx.TMax = m.setTimeout(rCtx)
 
-	m.metricEngine.RecordPublisherRequests(rCtx.Endpoint, rCtx.PubIDStr, rCtx.Platform)
+	m.metricEngine.RecordPublisherRequests(rCtx.Endpoint, rCtx.PubIDStr, getPubmaticPlatform(rCtx.Platform))
```
Comment (pm-viral-vala) [inline modules/pubmatic/openwrap/beforevalidationhook.go:108]: can we move these changes inside of `stats/tcp_stats.go#RecordPublisherRequests` ?

**Review by pm-viral-vala — CHANGES_REQUESTED** [review]

File: modules/pubmatic/openwrap/beforevalidationhook.go (hunk 1 above)
Comment (ashishshinde-pubm) [reply modules/pubmatic/openwrap/beforevalidationhook.go]: done

**Review by pm-viral-vala — APPROVED** [review]

**Review by pm-jaydeep-mohite — APPROVED** [review]
LGTM

**Review by pm-nilesh-chate — APPROVED** [review]

**Pull request merged**